example_configs/ejabberd.md

```diff
@@ -0,0 +1,30 @@
+# Basic LDAP auth for a Ejabberd XMPP server
+
+[Main documentation here.](https://docs.ejabberd.im/admin/configuration/ldap/)
+
+For simple user auth add this to main ejabberd.yml:
+
+```
+host_config:
+  xmpp.example.org:
+    auth_method: [ldap]
+    ldap_servers:
+      - 127.0.0.1
+    ldap_port: 3890
+    ldap_uids:
+      - uid
+    ldap_rootdn: "uid=admin,ou=people,dc=example,dc=org"
+    ldap_password: "secret"
+    ldap_base: "ou=people,dc=example,dc=org"
+```
+
+## vCard from LDAP
+Theoretically possible, [see the documentation.](https://docs.ejabberd.im/admin/configuration/ldap/#vcard-in-ldap)
+
+TODO
```

Comment:
The birthday is not supported yet. If it's required, then that's blocking on #67

Reply:
Thats all optional. I actually only plan to get the photo and display name from LDAP. But good to know that additional values are worked on.